projectm-sys/.cargo/config.toml

```diff
@@ -1,2 +1,5 @@
 [target.wasm32-unknown-emscripten]
 linker = ".cargo/linker-emscripten"
+
+[target.x86_64-unknown-linux-gnu]
+rustflags = "-C link-arg=-lGL -C link-arg=-lstdc++ -C link-arg=-lgomp"
```

Comment:
are these needed?
should we really overwrite linker args just for linux/x86?

Reply:
I've found that these are necessary otherwise the linker complains. I think rustc only links libc, not libc++, and gomp which I guess is a projectm dependency of some sort.